web/Views/Shared/_Layout.cshtml

```diff
@@ -26,6 +26,12 @@
                         <li class="nav-item">
                             <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
                         </li>
+                        @if (User.Identity?.IsAuthenticated == true)
+                         {
+                             <li class="nav-item">
+                                 <a class="nav-link text-dark" asp-area="" asp-controller="Landlord" asp-action="ViewProperties">View Properties</a>
+                             </li>
+                         }

```

Comment:
We can't find a way to check if a user is a landlord from this `_Layout` file (and I think @TomHanmer29 and @blancamc00 had a similar issue). For now, we show this link to any authenticated user, then if a non-landlord clicks the link they get a 403.

A possible solution would be to add a `Landlord` role, similar to the existing `Admin` one. We could then do something like `@if (User.IsInRole("Landlord"))` to selectively show navbar elements. But I thought this might warrant its own ticket.

Reply:
Alternatively, `((BricksAndHeartsUser) User.Identity).LandlordId` seems to work without any changes. Which option would be better here? The role approach might also be useful for `[Authorize(Roles = "Landlord")]` on other pages.

Reply:
Lets do `((BricksAndHeartsUser) User.Identity).LandlordId != null` for now.

Adding a new role sounds sensible, agreed it should be a separate PR at least, and may as well have a ticket, do raise one.